packages/meditrak-server/src/tests/questionHooks.test.js

```diff
@@ -251,6 +251,11 @@ describe('Question hooks', () => {
 
       const entity = await models.entity.findById(ENTITY_ID);
       expect(entity.image_url).to.equal(TEST_URL);
+
+      // should be otherwise unchanged
+      const { image_url: beforeImageUrl, ...beforeData } = await beforeEntity.getData();
+      const { image_url: afterImageUrl, ...afterData } = await entity.getData();
+      expect(beforeData).to.deep.equal(afterData);
```

Comment:
To get this test working I needed to add https://github.com/beyondessential/tupaia/pull/1412/files#diff-a7797673ad2fcb3534ce390d7d24f2b2R137, so I wanted to add a test here to make sure that wasn't destructive

Reply:
Good call!